setup.py

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env python3
 
+# Start ignoring PyImportSortBear and PycodestyleBear* as imports below may
+# yield syntax errors
+from coalib import assert_supported_version, VERSION, get_version
+
+assert_supported_version()
+
```

Comment:
According to the official repo `from coalib import assert_supported_version, VERSION, get_version` and `assert_supported_version()` are already at lines 20 and 30. `Start ignoring...` and `Stop ignoring...` are also removed.